misc/loop/tools.go

```diff
@@ -0,0 +1,6 @@
+package tools
+
+import (
+	_ "github.com/gnolang/gno/gno.land/cmd/gnoland"
+	_ "github.com/gnolang/tx-archive/cmd"
```

Comment:
what's the current status on considering to switch tx-archive in this repo?

Reply:
I am completely fine with moving the archiver to the monorepo, along with our `gnolang/faucet`.

Do you want me to open a PR that migrates it?

Reply:
Are there good reasons to keep our repo by itself? We could try keeping the Dockerfile in a different repo and leave the main parts and libraries here.

If we make our 'gno' repo depend on another, and that one also depends on 'gno', it might cause problems with versions not matching up and make it harder to manage when we update things.

Reply:
I don't think we lose anything by merging it into the monorepo - given that we keep the `go.mod` of the original repo that we migrate (just to keep imports separate, and not bloat the main gno `go.mod`), and retain some ability to do releases on this repo, which shouldn't be too tricky

What do you think @ajnavarro @gfanton ?